features/git-extract/cherry_pick_conflict/without_remote_origin.feature

```diff
@@ -38,10 +37,9 @@ Feature: git extract: resolving conflicts between main branch and extracted comm
     Then it runs the commands
       | BRANCH   | COMMAND                 |
       | refactor | git cherry-pick --abort |
-      |          | git checkout main       |
-      | main     | git branch -d refactor  |
       |          | git checkout feature    |
-      | feature  | git stash pop           |
+      | feature  | git branch -D refactor  |
```

Comment:
have to use force here as refactor isn't complete merged into HEAD, which is feature at the moment.